configs/ocp_is2re/dimenet.yml

```diff
@@ -0,0 +1,30 @@
+includes:
+- configs/ocp_is2re/base.yml
+
+model:
+  name: dimenet
+  hidden_channels: 128
+  num_blocks: 2
+  cutoff: 6.0
+  num_radial: 3
+  num_spherical: 4
+  num_before_skip: 1
+  num_after_skip: 2
+  num_output_layers: 3
+  use_pbc: True
+  regress_forces: False
+
```

Comment:
Were you able to train dimenet with this config file?

Reply:
Yeah, this trains just fine.

From my brief profiling, the SBF layer and fully-connected layers within each conv layer take up most of the time. Hence reduced `num_spherical` and `num_radial`, and the no. of conv layers. The above was about twice as fast than the default hyperparameters and gets similar accuracies for IS2RE.

Reply:
Awesome, good to know.